### DISCUSSION TITLE
how to appear on https://github.com/sponsors/explore?

### DISCUSSION THREAD

**Discussion by monperrus** [community/community#55497] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi Github,

When I look at  https://github.com/sponsors/explore?account=ASSERT-KTH I don't find INRIA/spoon, which is a sponsorable organization with FUNDING.yml.

Why? How to fix this?

Thanks!

--Martin

**Comment by anjuan** [comment]
Hi @monperrus. The [Explore page](https://github.com/sponsors/explore) uses the dependency graph to determine who you can sponsor. [INRA/spoons ](https://github.com/INRIA/spoon)is definitely [sponsorable via Open Collective](https://opencollective.com/spoon-java) through the FUNDING.yml file, but perhaps you don't have any repos that use them?

**Comment by monperrus** [comment]
> [INRA/spoons ](https://github.com/INRIA/spoon)is definitely [sponsorable via Open Collective](https://opencollective.com/spoon-java) through the FUNDING.yml file

yes, that's good.

> perhaps you don't have any repos that use them?

we do, see https://github.com/search?q=org%3AASSERT-KTH+spoon-core+language%3A%22Maven+POM%22&type=code&l=Maven+POM

for some reasons, Github does not consider them as sponsorable and does not propose spoon on https://github.com/sponsors/explore.

how to fix this?

Thanks @anjuan

**Comment by hbakri** [comment]
Hi @monperrus ! As @anjuan said, the [Github Sponsors Explore Page](https://github.com/sponsors/explore) include a list of projects your accounts depend on, _BUT_ it will only indicate which ones are eligible for **GitHub sponsorship** and therefore not external ones such as [Open Collective](https://opencollective.com/spoon-java).

<img width="449" alt="Capture d’écran 2023-07-12 à 17 31 49" src="https://github.com/community/community/assets/92298226/1b1390d3-4382-4d33-9874-0d1377a4a0b4">

But don't worry, you can easily become a sponsored developer by joining [GitHub Sponsors](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account).
 
Hope it helps you!

**Comment by monperrus** [comment]
@hbakri we added a sponsorable Github org in [FUNDING.yml](https://github.com/INRIA/spoon/blob/master/.github/FUNDING.yml)

yet, the project is still not recognised as sponsorable: <https://github.com/search?q=org%3AINRIA%20is%3Asponsorable&type=repositories>

how to fix this?

thanks for your help!
